### ISSUE TITLE
SingleChildScrollView scroll slow when put in Transform.scale

### ISSUE THREAD

**Issue by wanjm** [flutter/flutter#35001]
I have a big SingleChildScrollView , and then put it in Transform.scale to scale it to small size; but the scroll move slow than expected. I think SingleChildScrollView should responsible for this;
```import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: Scaffold(
        appBar: AppBar(
          title: Text("scroll"),
        ),
        body: LayoutBuilder(
          builder: builder,
        ),
      ),
    );
  }

  Widget builder(BuildContext context, BoxConstraints constraints) {
    var width = constraints.maxWidth;
    var height = constraints.maxHeight;
    return Scroll(
      width: width,
      height: height,
    );
  }
}

class Scroll extends StatelessWidget {
  double height;
  double width;

  Scroll({this.width, this.height}) : super();
  Widget build(BuildContext context) {
    var w = width * 5;
    var h = height * 5;
    var view = SingleChildScrollView(
      child: Container(
        width: w,
        height: h * 5,
        color: Colors.red,
        child: MyList(),
      ),
    );
    return Center(
      child: OverflowBox(
        maxWidth: w,
        minWidth: w,
        minHeight: h,
        maxHeight: h,
        child: Transform.scale(
          scale: 0.2,
          child: view,
        ),
      ),
    );
  }
}

class MyList extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    List<Widget> children = List<Widget>();
    for (var i = 0; i < 10; i++) {
      children.add(Container(
        child: Text(
          "content $i",
          style: TextStyle(fontSize: 100,color: Colors.green),
        ),
        height: 400,
        color: i % 2 == 0 ? Colors.black : Colors.blue,
      ));
    }
    return Column(
      crossAxisAlignment: CrossAxisAlignment.stretch,
      children: children,
    );
  }
}
```

**Comment by wanjm** [comment]
it is fixed in 1.9.1

**Comment by wanjm** [comment]
it is fixed in 1.9.1
